kano/network.py

```diff
@@ -551,7 +631,9 @@ def connect(iface, essid, encrypt='off', seckey=None, wpa_custom_file=None, conn
     and internet connection to become ready.
     '''
 
-    if os.access('/var/run/kano-connect.pid', os.R_OK):
+    wpafile=None
```

Comment:
Currently a little confusing being called the "wpa" file as it can actually be a file for both WPA and WEP connections. Maybe just call it something like `conf_file` or `wifi_conf_file`?

Reply:
Makes a lot of sense, done.